lib/simple-connect-policy.js

```diff
@@ -6,21 +6,21 @@
 // anonymous handshake.
 //
 module.exports = class SimpleConnectPolicy {
-  constructor(login, password, version) {
+  constructor(login, password) {
     this.login = login;
     this.password = password;
-    this.version = version;
   }
 
   // Should send handshake packet with appropriate credentials
   // You can get client object provided upon connection creation
   // with connection.client.
+  //   app - string or object, application to connect to as 'name' or
+  //         'name@version' or { name, version }
```

Comment:
You should specify the constraints for `version` like you did in `lib/connection.js` on lines 114-115.